docs/docs/multi-dc.md

```diff
@@ -0,0 +1,176 @@
+Multi Datacenter
+================
+
+As of version 0.2.0 Cassandra on Mesos allows for providing configuration
+to allow multiple datacenters to be aware of one another.
+
+
+## How it works
```

Comment:
I'd reword to: "As of Cassandra 0.2.0, you can configure Cassandra on Mesos so that multiple datacenters are aware of each other."